pyproject.toml

```diff
@@ -0,0 +1,29 @@
+[tool.poetry]
+name = "swodlr-ingest-to-sds"
+version = "0.0.1"
+description = "A system for ingesting input products for the SWOT SDS"
+authors = ["podaac-tva <podaac-tva@jpl.nasa.gov>"]
+license = "Apache-2.0"
+readme = "README.md"
+packages = [
+  { include = "podaac" }
+]
+
+[tool.poetry.dependencies]
+python = "^3.11"
```

Comment:
Is 3.11 required or can we relax this a bit to 3.9 or 3.10?

Reply:
3.8 seems reasonable actually https://endoflife.date/python
Unless we're doing something that really requires 3.11 but it doesn't look like it

Reply:
It was future proofing to be honest, but I can relax it back down to 3.9

Reply:
Changed in [5ecc20c](https://github.com/podaac/swodlr-ingest-to-sds/pull/1/commits/5ecc20ce8402eaddf0e9df0e1c87a6f532cfef43)